lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -107,7 +108,7 @@ const CurrencyExchangeRateFields = ({
           required={isExchangeRateRequired}
           validate={isExchangeRateRequired ? validateRequired : undefined}
           key={isExchangeRateRequired ? 1 : 0}
-          isNonInteractive={isNonInteractive}
+          isNonInteractive={isSetExchangeRateNonIntaractive ?? isNonInteractive}
```

Comment:
so from the business perspective does it make sense such situation when `isNonInteractive` is true (currency field disabled) and `isSetExchangeRateNonIntaractive` is `false` and you can change exchange rate?

Reply:
As I mention above. The case - invoice has the status `Approved` (the currency is non-interactive, the exchange rate could be changed if needed and becomes non-interactive if the invoice is paid)